benchmarks/build.gradle

```diff
@@ -47,7 +54,23 @@ tasks.named("compileJava").configure {
   options.compilerArgs.addAll(["-processor", "org.openjdk.jmh.generators.BenchmarkProcessor"])
 }
 
-tasks.named("run").configure { executable = "${BuildParams.runtimeJavaHome}/bin/java"}
+task copyExpression(type: Copy) {
+  dependsOn configurations.expression
+  from { configurations.expression.collect { zipTree(it) } }
+  into "${buildDir}/plugins/expression"
+}
+
+task copyPainless(type: Copy) {
+  dependsOn configurations.painless
+  from { configurations.painless.collect { zipTree(it) } }
+  into "${buildDir}/plugins/painless"
+}
+
+tasks.named("run").configure {
+  executable = "${BuildParams.runtimeJavaHome}/bin/java"
+  jvmArgs << "-Dplugins.dir=${buildDir}/plugins" << "-Dtests.index=${buildDir}/index"
+  dependsOn copyExpression, copyPainless
```

Comment:
```suggestion
  dependsOn "copyExpression", "copyPainless"
```

Reply:
I was wondering if there is a better way to do all this. We have some infrastructure around loading these things for clusters. But its not exactly the same.